Makefile

```
@@ -0,0 +1,109 @@
+.PHONY: setup test lint format security clean docker-build docker-up docker-down
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)